### PR TITLE
Redesign Settings Pages, More functionality#381

### LINKED ISSUE #381: Redesign Settings Pages, More functionality
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
Current UI needs to be changed for settings page.

Theme color pick right now only allows 4 colors
screenshot 2018-09-29 at 1 36 41 pm

Each field in settings has a separate 'Submit' button.

Settings page is only accessible from the options tab in chrome(right click on extenison icon -> options) to access the settings

**Expected behavior:**

- Color picker should be like this:
screenshot 2018-09-29 at 1 35 17 pm

 - Only 1 submit button for the whole setting.

 - Settings page accessible from inside the app, that is, from the menu bar.

- Fix the layout, orientation of the settings page.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #381 

PR: https://github.com/fossasia/susi_chromebot/pull/388 had undetectable merge conflicts, so creating a new PR.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Screenshots:
screenshot 2018-09-30 at 9 28 17 pm
screenshot 2018-09-30 at 9 28 24 pm
screenshot 2018-09-30 at 9 32 56 pm

Before:
screenshot 2018-09-30 at 9 15 37 pm

#### Changes proposed in this pull request:

- Added settings in tab
- Modified settings page
- Added custom theme picker
- Add ignore DS_store file to gitignore

### CODE REVIEW

**Comment by fragm3** [comment]
@zamhaq @Orbiter @gabru-md @ms10398 @rupav please review.

**Comment by fragm3** [comment]
@gabru-md  The color picker used is external dependency, the CSS files wont make it slower, since its just styling.  And will be loaded in the settings page only, so maybe it's fine? Thanks for your review.

**Comment by fragm3** [comment]
@rupav You gave the wrong link.

The web app order:
screenshot 2018-10-08 at 5 12 18 pm

How about we order like this:
- Chat
- Skills
- Clear Chat
- Export
- Settings
- About
- Logout

File: src/css/options.css [hunk 1]
```diff
@@ -1,30 +1,34 @@
 body {
     background-color: #e8f2fe;
```
Comment (rossoskull) [inline src/css/options.css]: I think this should be removed

File: src/css/options.css [hunk 2]
```diff
@@ -1,30 +1,34 @@
...
-    color: #fff;
+    background-color: #F2F2F2;
+    color: rgba(0, 0, 0, 0.6);
 }
 
 .container{
     margin-top: 5px;
```
Comment (rossoskull) [inline src/css/options.css]: Same with this

File: src/css/options.css [hunk 3]
```diff
@@ -35,18 +39,62 @@ body {
...
+    float: right;
+}
+
+.blue-container{
+    background-color: #0084ff;
+    padding: 30px 30px 30px;
+    box-shadow: 2px 2px 5px rgba(0,0,0,0.5);
+    margin-bottom: 5px;
+    padding-bottom: 10px;
+    padding-top: 10px;
```
Comment (rossoskull) [inline src/css/options.css]: You can add this padding-bottom and padding-top in the padding style on line no. 64

**Comment by fragm3** [comment]
screenshot 2018-10-16 at 3 12 09 pm

Merge conflicts due to renaming of file. After renaming:
screenshot 2018-10-16 at 3 16 59 pm

**Pull request closed without merging**